mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/api/akka/MasterApiAkkaService.java

```diff
@@ -100,7 +103,7 @@ public MasterApiAkkaService(final MasterMonitor masterMonitor,
                                 final ResourceClusters resourceClusters,
                                 final ActorRef resourceClustersHostManagerActor,
                                 final int serverPort,
-                                final IMantisStorageProvider mantisStorageProvider,
+                                final MantisStorageProvider mantisStorageProvider,
```

Comment:
I would prefer we don't make these changes in this diff - @hmitnflx is working on cleaning this abstraction.

Reply:
If the other PR is ready to go I'm fine to rebase.

Reply:
spoke to @hmitnflx and he needs more time for testing the change. I will go head and merge this for now.